docs/source/getstarted.rst

```diff
@@ -59,109 +38,61 @@ Install LeApp-To using these steps: ::
     sudo dnf copr enable evilissimo/leapp
     sudo dnf install leapp-cockpit
 
-Optional (Demo setup): ::
-        
-    sudo dnf install vagrant qemu libguestfs-tools-c libvirt libvirt-devel ruby-devel gcc qemu-kvm nmap libffi-devel
-
-
-Preparation & Running the demo
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-If the integration tests haven't been run, first install the testing
-instance of the CLI: ::
-
-    pipsi --bin-dir $PWD/bin install --python `which python2.7` $PWD/src
-
-The integration tests do this automatically, so this step can be skipped if
-those have already been run.
-
-Make sure the libvirt daemon is running and enabled: ::
-
-    for i in start enable; do sudo systemctl $i libvirtd; done
+    
+Enable and Start Docker & Cockpit
+---------------------------------
 
-The prototype requires Vagrant with some relevant plugins: ::
+    systemctl enable docker
+    systemctl enable cockpit
+    systemctl start docker
+    systemctl start cockpit
 
-    sudo vagrant plugin install ansible hitimes nio4r vagrant-libvirt
 
-And passwordless access to VM management operations: ::
+Importing apps via the CLI
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    sudo usermod -aG vagrant,libvirt $USER
+First add your private key to your ssh-agent to enable passwordless authentificatoin::
```

Comment:
First add your private key for accessing the source machine as root to your running ssh-agent to enable passwordless authentication